src/firefly/js/templates/fireflyviewer/FireflyViewerManager.js

```diff
@@ -131,8 +133,9 @@ function handleLayoutChanges(action) {
     }
 }
 
-function handleNewTable(action, images, showImages) {
+function handleNewTable(action, images, showImages, showTables) {
```

Comment:
This saga is really confusing.  I plan to re-write it one day.  So, ignore the awkwardness for now.